src/api/nft.ts

```diff
@@ -71,4 +77,7 @@ export interface Nft extends BaseNft {
 
   /** URIs for accessing the NFT's media assets. */
   media: Media[];
+
```

Comment:
Can you add the `spamInfo` field to `OwnedNft` (https://github.com/alchemyplatform/alchemy-sdk-js/blob/ba8ce861b3da5a3e1dc7635b21f5da9d4137d66a/src/types/types.ts#L569) instead? That way the field is only present in `getNftsForOwner()` and not `getNftsForContract()`.

Reply:
spamInfo is included in the response of getNftsForContract as well (it's a bit repetitive admittedly).

We also can't always extract this field from the raw NFT the way we do with `balance` because spamInfo is only present when we've fetched metadata (i.e. it's not present on any of the Base* types)

Reply:
I don't see it here: https://eth-mainnet.g.alchemy.com/nft/v2/demo/getNFTsForCollection?contractAddress=0x01234567bac6ff94d7e4f0ee23119cf848f93245&withMetadata=true

Are we planning on releasing this soon, or is it only on certain contracts?

Reply:
ah yup yup it's only present for spam contracts